client/lib/domains/whois/test/assembler.js

```diff
@@ -6,105 +6,144 @@ const expect = require( 'chai' ).expect;
 /**
  * Internal dependencies
  */
-const whoisAssembler = require( './../assembler' );
+const whoisAssembler = require( './../assembler' ),
+	{ whoisType } = require( './../constants' );
+
+const FIRST_NAME = 'First-name',
+	LAST_NAME = 'Last-name',
+	ORGANIZATION = 'Organisation',
+	ADDRESS = 'Address line 1',
+	ADDRESS_LINE_2 = 'Address line 2',
+	CITY = 'City',
+	STATE_CODE = 'ST',
+	STATE = 'State',
+	POSTAL_CODE = '12345',
+	COUNTRY_CODE = 'CO',
+	COUNTRY = 'Country',
+	EMAIL = 'email@email.com',
+	PHONE = '123456789',
+	FAX = '987654321';
+
+function buildEmptyWhois( type ) {
+	return {
+		firstName: '',
+		lastName: '',
+		organization: '',
+		address1: '',
+		address2: '',
+		city: '',
+		state: '',
+		stateName: '',
+		postalCode: '',
+		countryCode: '',
+		countryName: '',
+		email: '',
+		phone: '',
+		fax: '',
+		type
+	};
+}
+
+function buildWhois( suffix, type ) {
+	return {
+		firstName: FIRST_NAME + suffix,
+		lastName: LAST_NAME + suffix,
+		organization: ORGANIZATION + suffix,
+		address1: ADDRESS + suffix,
+		address2: ADDRESS_LINE_2 + suffix,
+		city: CITY + suffix,
+		state: STATE_CODE + suffix,
+		stateName: STATE + suffix,
+		postalCode: POSTAL_CODE + suffix,
+		countryCode: COUNTRY_CODE + suffix,
+		countryName: COUNTRY + suffix,
+		email: EMAIL + suffix,
+		phone: PHONE + suffix,
+		fax: FAX + suffix,
+		type
```

Comment:
Very minor nit-pick here: 
According to the guidelines, type should be placed at the [beginning of the object](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#object-property-shorthand)